runtime/dali/Cargo.toml

```diff
@@ -197,7 +198,7 @@ std = [
 	"crowdloan-rewards/std",
 	"bonded-finance/std",
 	"dutch-auction/std",
-	"liquidations/std",
+	"lending/std",
```

Comment:
Add `liquidations/std`?

Reply:
is is lower in the list. "liquidations/std", it was actually double added, removed on entry